interpreter/python/decode_amd64.go

```diff
@@ -1,30 +1,163 @@
-//go:build amd64
-
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
 package python // import "go.opentelemetry.io/ebpf-profiler/interpreter/python"
 
 import (
-	"unsafe"
+	"fmt"
 
 	"go.opentelemetry.io/ebpf-profiler/libpf"
-	_ "go.opentelemetry.io/ebpf-profiler/zydis" // links Zydis
+	"golang.org/x/arch/x86/x86asm"
 )
 
-// #cgo CFLAGS: -g -Wall
-// #include "decode_amd64.h"
-// #include "../../support/ebpf/types.h"
-import "C"
+const debugDecodeAMD64 = false
```

Comment:
I would prefer to drop this. The debug prints that are not important can be dropped, and the meaningful ones should use the same logging mechanism as other code. It was `#ifdef` in the C code only because adding interface to the logger from C was not found important enough.